src/store/modules/room.ts

```diff
@@ -108,26 +84,26 @@ const RoomModule: Module<RoomState, {}> = {
         name: GameName['NONE'],
         api: []
       },
-      collections: [],
-      tactics: [],
-      users: [],
-      isPresentationEnabled: false,
-      presentationEnabledBy: undefined,
-      isPrivate: false
+      presentation: {
+        enabledBy: undefined,
+        tacticId: undefined
+      },
+      isPrivate: false,
+      isCanvasLoaded: false,
```

Comment:
i think isCanvasLoaded should be user specific, not room specific

Reply:
i'll move it to a new module as discussed in discord

Reply:
I'll do this as part of another ticket this weekend